src/components/ui/SROnly/SROnly.tsx

```diff
@@ -1,13 +1,17 @@
+import type { ElementType } from 'react'
 import React from 'react'
 
 import './sr-only.scss'
 
 interface Props {
   text: string
+  as?: ElementType
 }
 
-function SROnly({ text }: Props) {
-  return <span data-store-sr-only>{text}</span>
+function SROnly({ text, as }: Props) {
```

Comment:
In the future we can rename it to a more generic meaning to cover this case (when we need to hide information but not necessary related to screen reader only)  👍